templates/SideBySide/index.html.twig

```diff
@@ -32,15 +32,15 @@
     <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/file-uploader/5.11.8/jquery.fine-uploader/jquery.fine-uploader.min.js"></script>
     <script type="text/javascript" src="//maps.google.com/maps/api/js?v=3&key={{ google_maps_api_key }}"></script>
 
-    <script type="text/javascript" src="{{ asset('js/pelagosNoty.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/datasetReview.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/diff_match_patch.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/sidebyside.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/geoviz.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/mapWizard.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/gMap.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/common.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/fileBrowser.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/pelagosNoty.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/datasetReview.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/diff_match_patch.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/sidebyside.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/geoviz.js') }}"></script>
```

Comment:
Geoviz needs to be moved too

Reply:
yeah, I only matched everything in the templates directory...thanks for the catch!